src/how-to-stacker.md

```diff
@@ -1 +1,72 @@
 # How to Participate as a Stacker
+Participating as a Stacker in the Stacks Blockchain is an essential role that involves validating and adding new sBTC transactions to the blockchain. As a Stacker, you are responsible for ensuring the accuracy and integrity of these transactions. To become a Stacker, you must hold and temporarily lock STX, Stacks’ native currency, and support the network’s security and consensus. As a reward, you earn BTC. 
+
+## Introduction
```

Comment:
I'd consider the paragraph above also part of the introduction so we can skip this header.
```suggestion
```